governator-core/src/main/java/com/netflix/governator/annotations/AutoBind.java

```diff
@@ -27,6 +27,8 @@
  * Annotation binding that combines with Governator's classpath scanning and a bound
  * {@link AutoBindProvider} to automatically/programmatically bind fields and constructor/method
  * arguments
+ * 
+ * @deprecated  AutoBind is deprecated in favor of using Guice modules
  */
 @Documented
```

Comment:
It is probably a good idea to use the `@Deprecated` annotation as well to guarantee a warning will be shown and also improve detection of deprecation with other tools.